packages/dialog/src/vaadin-dialog.js

```diff
@@ -266,20 +223,6 @@ class Dialog extends OverlayClassMixin(
     }
   }
 
-  /** @protected */
-  disconnectedCallback() {
```

Comment:
The `connectedCallback` can also be removed here as its implementation is now in the mixin.

Reply:
Thanks, removed.